### PR TITLE
feat: added badge component

### PR DESCRIPTION
**Pull request by mansi2011**
feat: added badge component
- badge on icon
- badge on avatar

badge

### CODE REVIEW

**Review by Manpreet-hub — APPROVED** [review]
code looks good.

File: Components/Badge/Badge.html [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+                    </div>
+                </div>
+                <div class="avatar-badge-item">
+                    <div class="user"><img class="small-avatar" src="/Resources/avatar1.jpg">
+                        <span class="status status-dnd">.</span></div>
+                    
+                </div>
+                <div class="avatar-badge-item">
+                    <div class="user">
+                        <img class="small-avatar" src="/Resources/avatar1.jpg">
```
Comment (devananth) [inline Components/Badge/Badge.html:94]: `alt` attribute is missing in `img` tag

**Review by devananth — COMMENTED** [review]
Everything else is fine.

File: Components/Badge/Badge.html (hunk 1 above)
Comment (mansi2011) [reply Components/Badge/Badge.html:94]: added now thanks

**Pull request merged**